deb/debian/rules

```diff
@@ -20,6 +22,11 @@ ifeq ($(DISTRO),xenial)
 DOCKER_FLAG="--enable-docker"
 endif
 
+ifeq ($(DISTRO),bionic)
+ENABLE_MONGO_FLAG=""
```

Comment:
I would combine these into a single `MONGO_FLAG`.

Reply:
Just pushed a change to combine these flags into a single variable